troxy-server/src/main/java/no/sb1/troxy/util/SimulatorHandler.java

```diff
@@ -232,9 +232,7 @@ public void handle(String target, org.eclipse.jetty.server.Request jettyRequest,
         byte[] contentBytes = response.getContent().getBytes(response.discoverCharset());
         /* status */
         try {
-            int code = Integer.parseInt(response.getCode());
-            if (code < 400) servletResponse.setStatus(code);
-            else servletResponse.sendError(code, response.getReason());
```

Comment:
Foreslår heller at servletResponse.sendError(code, response.getReason()) endres til 

servletResponse.setStatus(code, response.getReason());

Reply:
Suggest we use this fix instead:

https://github.com/SpareBank1/Troxy/pull/27

Reply:
Då denna fixas i andra pullrequester så tar jag bort denna